src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -376,7 +374,7 @@ private async Task SelectAndStartTransport(TransferFormat transferFormat, Cancel
                         continue;
                     }
 
-                    if (transportType == HttpTransportType.ServerSentEvents && _isRunningInBrowser)
+                    if (transportType == HttpTransportType.ServerSentEvents && RuntimeInformation.IsOSPlatform(OSPlatform.Create("browser")))
```

Comment:
This doesn't seem right. Why aren't we using the new `OperationSystem.*` API?

Reply:
It's only available on net5 which makes it a lot messier to use.

Reply:
Can't we just add a helper on the other platforms? The linker recognizes that pattern doesn't it?

Reply:
That might be an option. Let me see what that looks like.